src/styles/block/catalog.scss

```diff
@@ -0,0 +1,15 @@
+.container {
+  --colum-count: 1;
+
+  justify-content: center;
+  align-items: center;
+  display: grid;
+  padding: 50px 40px;
+
+  grid-template-columns: repeat(var(--colum-count), 200px);
+  gap: 46px 48px;
+}
+
+@include on-phone;
+@include on-tablet;
+@include on-pc;
```

Comment:
It seems you're trying to include mixins for responsive design, but you haven't passed any content to them. According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), mixins should be used with content blocks to apply styles for different devices. You need to provide the corresponding styles inside these mixins.

Reply:
Good comment. Now mixins aren't reusable (you don't need to reuse them in this task, but anyway it looks a bit strange). You can write these mixins inside of the `.container` selector and apply correct styles in each case, schematically:

```
.container {
  @include on-phone {
    --column-count: 2;
  }
}
```
And inside each mixin definition use `@content`